electron/src/IPCEvents/G14ControlPlans.ts

```diff
@@ -17,6 +17,7 @@ import {
 import { setRyzenadj } from './ryzenadj/ModifyCPU';
 
 const LOGGER = getLogger('G14ControlPlans');
+export let current_plan = null
```

Comment:
I'm not a big fan of how I maintain this state, my typescript is a bit rusty.. Please tell me if we can do better here :)